### ISSUE TITLE
12.3 Redirects

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3748]
### Summary

**What we are after:**
_In a few of our content tickets this sprint, we have made new pages or uploaded documents to the CMS that will replace transition files. This ticket is to put in the redirects for them._

### Related issues

- #3727 - Enforcement materials
- #3724 - Revamped disclaimer page

### Completion criteria

- [x] Redirects put in for disclaimer pages and enforcement materials files (see PI12 page of [this spreadsheet](https://docs.google.com/spreadsheets/d/1Jn5nC3fFku10RTCP5OAW6RiffjNBE4VwBubITQpCNK8/edit?usp=sharing)
- [x] Add files to be taken down to the issue at #3609

### Future work

- Front end removes these files from transition (See #3609 )

**Comment by djgarr** [comment]
PR submitted at https://github.com/fecgov/fec-proxy/pull/165

**Comment by dorothyyeager** [comment]
Because of the lengthy list and wanting to train @djgarr on how to do redirects, we split them into two different bunches (David did disclaimer pages and OGC enforcement files; I did RAD and Audit enforcement files) and put in transition and www redirects for each bunch. We tested being able to do Wagtail redirects, but it only worked for one page that was replacing another Wagtail page. @patphongs We've tagged you as a reviewer and @kathycarothers and I will review for content team.
